crypto/armv4cpuid.pl

```diff
@@ -293,6 +293,7 @@
 #endif
 
 .extern	OPENSSL_armcap_P
+.hidden	OPENSSL_armcap_P
```

Comment:
Rather than making an identical change in every Perl-asm file, and then requiring this same pairing in every new Perl-asm that gets added in the future, wouldn't it be better to have `arm-xlate.pl` apply this directly? (At least, that's the way I've been looking at fixing this)

Reply:
I can't say much about openssl's perl asm stuff. 
First, it should be validated that `.hidden` is the right thing here.

Reply:
Note that this isn't what BoringSSL has anyway - they still have `.comm` rather than `.extern`. One of our recent-ish changes made that here, but I am now revisiting whether that was the best thing to do, as `arm-xlate.pl` actually removes the `.extern` altogether, but `.comm` keeps it in